src/app/trips/create-trip/create-trip.component.html

```diff
@@ -1,9 +1,313 @@
-<div class="container-fluid">
-  <h1>Create Trip</h1>
-
-  <h3> [ FORM FOR CREATING A TRIP WILL BE CREATED SOON... ]</h3>
-  <br/>
-  <br/>
-  <button type="submit" class="btn btn-primary">Create</button>
-  <button type="submit" class="btn btn-default" (click)="cancel()">Cancel</button>
+<!-- background image -->
```

Comment:
This is a very long component. You should look for opportunities to break it up into Angular components. Individual forms can be separated out and reused.